src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -104,13 +107,46 @@ const keriAcdcChangeHandler = async (
     dispatch(setCurrentOperation(OperationType.IDLE));
   }
 };
-
+const ACTIVTY_TIMEOUT = 14000;
 const AppWrapper = (props: { children: ReactNode }) => {
   const dispatch = useAppDispatch();
   const authentication = useAppSelector(getAuthentication);
   const [initialised, setInitialised] = useState(false);
   const [agentInitErr, setAgentInitErr] = useState(false);
 
+  useEffect(() => {
+    let timer: NodeJS.Timeout;
+    const handleActivity = () => {
+      clearTimeout(timer);
+      timer = setTimeout(() => {
+        dispatch(logout());
+      }, ACTIVTY_TIMEOUT);
```

Comment:
Initially I had created the story to only target when the app was in the background/device locked etc. I am OK with it also happening when the app is open but for me it needs to be longer when the app is not in the background.

Can we detect those background states? You mentioned something about this this morning I think but can't fully remember.